radar-upload-backend/src/main/java/org/radarbase/upload/doa/RecordRepositoryImpl.kt

```diff
@@ -79,26 +79,26 @@ class RecordRepositoryImpl(@Context private var em: EntityManager) : RecordRepos
     override fun updateContent(record: Record, fileName: String, contentType: String, stream: InputStream, length: Long): RecordContent = em.transact {
         val existingContent = record.contents?.find { it.fileName == fileName }
 
-        val result = if (existingContent == null) {
-            val newContent = RecordContent().apply {
-                this.record = record
-                this.fileName = fileName
-                this.createdDate = Instant.now()
-                this.contentType = contentType
-                this.size = length
-                this.content = Hibernate.getLobCreator(em.unwrap(Session::class.java)).createBlob(stream, length)
-            }
-            record.contents = mutableSetOf(newContent)
-            newContent
-        } else {
-            existingContent.apply {
-                this.createdDate = Instant.now()
-                this.contentType = contentType
-                this.content = Hibernate.getLobCreator(em.unwrap(Session::class.java)).createBlob(stream, length)
+        refresh(record)
```

Comment:
This breaks the integration test somehow. I am not sure how it works for the doa test, but not for the integration test.
```
18:26:54.164 [grizzly-http-server-1] ERROR org.radarbase.upload.exception.UnhandledExceptionMapper - [500] http://0.0.0.0:8080/radar-upload/records/95/contents/TEST_ACC.csv
javax.persistence.RollbackException: Error while committing the transaction
	at org.hibernate.internal.ExceptionConverterImpl.convertCommitException(ExceptionConverterImpl.java:81)
	at org.hibernate.engine.transaction.internal.TransactionImpl.commit(TransactionImpl.java:107)
	at org.radarbase.upload.inject.DoaEntityManagerFactoryKt.transact(DoaEntityManagerFactory.kt:62)
	at org.radarbase.upload.doa.RecordRepositoryImpl.updateContent(RecordRepositoryImpl.kt:78)
	at org.radarbase.upload.resource.RecordResource.putContents(RecordResource.kt:144)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory.lambda$static$0(ResourceMethodInvocationHandlerFactory.java:52)
	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:124)
	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:167)
	at org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$ResponseOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:176)
	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:79)
	at org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:469)
	at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:391)
	at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:80)
	at org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:253)
	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:248)
	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:244)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:292)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:274)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:244)
	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:265)
	at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:232)
	at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:679)
	at org.glassfish.jersey.grizzly2.httpserver.GrizzlyHttpContainer.service(GrizzlyHttpContainer.java:353)
	at org.glassfish.grizzly.http.server.HttpHandler$1.run(HttpHandler.java:200)
	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:569)
	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:549)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: javax.ws.rs.ProcessingException: Error resetting the buffered message content input stream.
	at org.glassfish.jersey.message.internal.EntityInputStream.reset(EntityInputStream.java:116)
	at org.hibernate.engine.jdbc.BlobProxy.resetIfNeeded(BlobProxy.java:70)
	at org.hibernate.engine.jdbc.BlobProxy.getUnderlyingStream(BlobProxy.java:63)
	at org.hibernate.engine.jdbc.BlobProxy.getStream(BlobProxy.java:59)
	at org.hibernate.engine.jdbc.BlobProxy.getBinaryStream(BlobProxy.java:120)
	at org.postgresql.jdbc.PgPreparedStatement.setBlob(PgPreparedStatement.java:1125)
	at org.hibernate.type.descriptor.sql.BlobTypeDescriptor$4$1.doBind(BlobTypeDescriptor.java:132)
	at org.hibernate.type.descriptor.sql.BasicBinder.bind(BasicBinder.java:74)
	at org.hibernate.type.AbstractStandardBasicType.nullSafeSet(AbstractStandardBasicType.java:276)
	at org.hibernate.type.AbstractStandardBasicType.nullSafeSet(AbstractStandardBasicType.java:271)
	at org.hibernate.type.AbstractSingleColumnStandardBasicType.nullSafeSet(AbstractSingleColumnStandardBasicType.java:39)
	at org.hibernate.persister.entity.AbstractEntityPersister.dehydrate(AbstractEntityPersister.java:2875)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3172)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3696)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:90)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:604)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:478)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:356)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:39)
	at org.hibernate.internal.SessionImpl.doFlush(SessionImpl.java:1484)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:515)
	at org.hibernate.internal.SessionImpl.flushBeforeTransactionCompletion(SessionImpl.java:3355)
	at org.hibernate.internal.SessionImpl.beforeTransactionCompletion(SessionImpl.java:2526)
	at org.hibernate.engine.jdbc.internal.JdbcCoordinatorImpl.beforeTransactionCompletion(JdbcCoordinatorImpl.java:473)
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl.beforeCompletionCallback(JdbcResourceLocalTransactionCoordinatorImpl.java:178)
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl.access$300(JdbcResourceLocalTransactionCoordinatorImpl.java:39)
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.commit(JdbcResourceLocalTransactionCoordinatorImpl.java:271)
	at org.hibernate.engine.transaction.internal.TransactionImpl.commit(TransactionImpl.java:104)
	... 29 common frames omitted
Caused by: java.io.IOException: Mark not set
	at org.glassfish.grizzly.http.io.InputBuffer.reset(InputBuffer.java:648)
	at org.glassfish.grizzly.http.server.NIOInputStreamImpl.reset(NIOInputStreamImpl.java:94)
	at org.glassfish.jersey.message.internal.EntityInputStream.reset(EntityInputStream.java:114)
	... 56 common frames omitted
```